torch/onnx/_internal/fx/export.py

```diff
@@ -435,17 +532,34 @@ def _export(
     # Export FX graph to ONNX ModelProto.
     if decomposition_table is None:
         # Use default decomposition table.
-        decomposition_table = torch._decomp.decomposition_table
+        decomposition_table = _ONNX_FRIENDLY_DECOMPOSITION_TABLE
     # Apply decomposition table to the input graph.
-    decomposed_module = proxy_tensor.make_fx(module, decomposition_table)(*args)
-
-    decomposed_module = shape_inference_with_fake_tensor(decomposed_module, *args)
-
-    ts_graph, ts_initializers = _export_fx_to_ts(decomposed_module, opset_version)
-    # Export TorchScript graph to ONNX ModelProto.
-    onnx_model = _ts_graph_to_onnx_model_in_protobuf(
-        ts_graph, ts_initializers, opset_version
-    )
+    # Make sure the feed-in "module" is stateless.
```

Comment:
> Make sure

Is this a requirement on "module", or is this stating that the following `make_fx` call checks (and converts) "module" as 'stateless'?

Reply:
Yes, on "module". Otherwise, `make_fx` may fail. You can see in aot_autograd.py, stateful callable (i.e., nn.Module) and function are handled differently before calling into `make_fx`.